Source/DafnyCore/DooFile.cs

```diff
@@ -216,7 +215,8 @@ public class ManifestData {
       result.Options.OptionArguments[option] = libraryValue;
       result.ApplyBinding(option);
       var prefix = $"cannot load {options.GetPrintPath(libraryFile.LocalPath)}";
-      success = success && check(reporter, origin, prefix, option, localValue, libraryValue);
+      var checkpasses = OptionRegistry.GlobalCheck(option)?.Invoke(reporter, origin, prefix, option, localValue, libraryValue) ?? true;
```

Comment:
The CI warns that `libraryValue` is possibly null, is that intended?

Reply:
Fixed